testing/tests/filters.rs

```diff
@@ -122,7 +122,7 @@ mod filters {
     }
     // for test_nested_filter_ref
     pub fn mytrim(s: &dyn (::std::fmt::Display)) -> ::askama::Result<String> {
-        let s = format!("{}", s);
+        let s = format!("{s}");
         Ok(s.trim().to_owned())
```

Comment:
Alternatively:

```suggestion
        Ok(s.to_string().trim().to_owned())
```

Reply:
Since it's just a test, I have no opinion either way. :) The one line is a bit more idiomatic.

Reply:
Nice, squashed that into my change!